src/components/ui/Button/Button.tsx

```diff
@@ -16,15 +16,15 @@ export type ButtonProps = {
     props?: any
 }
 
-const Button = ({children, type='button', customRootClass='', color = '', className='', variant='solid', ...props}: ButtonProps) => {
+const Button = ({children, type='button', customRootClass='', color = '', className='', variant='solid',...props}: ButtonProps) => {
```

Comment:
let's add a size prop here, initialize it with an empty string for starters

- based on the size prop that is received here, we just pass it to the data-attribute

so we end up with
`data-size={size}`

once you render this in browser, this attribute should show up on the button, which you can use to style the buttons